arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -0,0 +1,740 @@
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryForLogSpecification": {
+      "description": "Log specification for the operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the log.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Localized display name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "blobDuration": {
+          "description": "Blob duration.",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryForServiceSpecification": {
+      "description": "Operation properties.",
+      "type": "object",
+      "properties": {
+        "logSpecifications": {
+          "description": "List of log specifications of this operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientDiscoveryForLogSpecification"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryProperties": {
+      "description": "Operation properties.",
+      "type": "object",
+      "properties": {
+        "serviceSpecification": {
+          "$ref": "#/definitions/ClientDiscoveryForServiceSpecification",
+          "description": "Operation properties.",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryResponse": {
+      "description": "List of available operations.",
+      "type": "object",
+      "properties": {
+        "Value": {
+          "description": "List of available operationss.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientDiscoveryValueForSingleApi"
+          },
+          "readOnly": true
+        },
+        "NextLink": {
+          "description": "NextLink",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClientDiscoveryValueForSingleApi": {
+      "description": "Available operation details.",
+      "type": "object",
+      "properties": {
+        "Name": {
+          "description": "Name of the operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Display": {
+          "$ref": "#/definitions/ClientDiscoveryDisplay",
+          "description": "Contains the localized display information for this particular operation.",
+          "readOnly": true
+        },
+        "Origin": {
+          "description": "The intended executor of the operation.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Properties": {
+          "$ref": "#/definitions/ClientDiscoveryProperties",
+          "description": "Properties of the operation.",
+          "readOnly": true,
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "Resource": {

```

Comment:
You can set move the ```x-ms-azure-resource``` extension from "TrackedResource" to "Resource". The extension does not determine if a resource is "TrackedResource" or not, the required property ```location``` does that